components/namespace-lister/base/kustomization.yaml

```
@@ -6,11 +6,12 @@ resources:
 - rbac.yaml
 - service.yaml
 - network_policy.yaml
+- ./metrics/
```

very minor, should we remove the `./` here?
```suggestion
- metrics/
```